menu/drivers/xmb.c

```diff
@@ -2359,6 +2359,9 @@ static uintptr_t xmb_icon_get_id(xmb_handle_t *xmb,
                      return xmb->textures.list[XMB_TEXTURE_RECORD];
                   case MENU_ENUM_LABEL_ONSCREEN_DISPLAY_SETTINGS:
                      return xmb->textures.list[XMB_TEXTURE_OSD];
+#ifdef HAVE_QT
+                  case MENU_ENUM_LABEL_SHOW_WIMP:
```

Comment:
this doesn't make sense... you're returning an xmb texture if the label is "show wimp" ?

Reply:
these are the icons besides the labels...

Reply:
ahh my apologies, didn't realize